.github/workflows/codeql-analysis-macos.yml

```diff
@@ -0,0 +1,49 @@
+name: "CodeQL for MacOS"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '23 11 * * 6'
+
+jobs:
+  analyze-macos:
+    name: Analyze MacOS (Catalina 10.15)
+    runs-on: macos-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp' ]
+        #language: [ 'cpp', 'javascript', 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more:
+        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install Qt
```

Comment:
You could install Qt via brew too: https://jamulus.io/wiki/Compiling#requirements I don't really trust this action since it might break in future. brew shouldn't.

Reply:
I've never used `brew`.  What is it?

Reply:
It's like apt for macOS: https://brew.sh/

Reply:
I've never used `choco` either...  Makes me wonder if there's an equivalent there.